sql/maintenance_utils.sql

```diff
@@ -44,6 +44,10 @@ CREATE OR REPLACE FUNCTION @extschema@.decompress_chunk(
     if_compressed BOOLEAN = true
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_decompress_chunk' LANGUAGE C STRICT VOLATILE;
 
+CREATE OR REPLACE PROCEDURE @extschema@.merge_chunks(
```

Comment:
I put the function i the extension schema instead of `_timescaledb_functions` since it is a function in line with `compress_chunk`, etc. But we could move it to `_timescaledb_functions` if we want this to be less of a public feature. :shrug:

Reply:
I think this is on the same level as `compress_chunk`, `drop_chunk`, and friends so we should keep it at the same place as these.